client/keystore/src/local.rs

```diff
@@ -0,0 +1,498 @@
+use std::{
```

Comment:
Missing license header.

Reply:
Not sure why github is not picking up the change here... but the license header has been added:
https://github.com/rakanalh/substrate/blob/async-keystore-auth-discovery/client/keystore/src/local.rs